Source/Core/VideoCommon/OpcodeDecoding.cpp

```diff
@@ -193,13 +193,7 @@ class RunCallback final : public Callback
   }
   OPCODE_CALLBACK(void OnUnknown(u8 opcode, const u8* data))
   {
-    if (static_cast<Opcode>(opcode) == Opcode::GX_UNKNOWN_RESET)
-    {
-      // Datel software uses this command
-      m_cycles += 6;
```

Comment:
For datel, we don't jump 6 cycles anymore?  Is this intentional?  Does it break anything?

Reply:
I believe we've measured these as 0 cycles on console

Reply:
I didn't do any timing measurement, but this results in command `0x01` (which seems to be invalid, and as far as I can tell is only accidentally used by datel, though 0x01 does also appear in the wiggler capsule vertex data) being treated the same as all other unknown commands (which are set to take 1 cycle, for some reason).

The no-op command (0x00) is currently set at 6 cycles, with a note that that may be too slow:

https://github.com/dolphin-emu/dolphin/blob/237947e2f139a033da71e733c5e6e24d61b5142d/Source/Core/VideoCommon/OpcodeDecoding.cpp#L190-L193

In any case, this change doesn't seem to impact datel titles (at least the ones I've tested), and all of these numbers are probably something that should be changed later if someone does better timing tests (I don't plan on doing them anytime soon).

Reply:
So, it's still 6 cycles regardless, gotcha.

Reply:
No, it does result in a difference (now it's 1 cycle while before it was 6 cycles), but it shouldn't affect anything other than situations that previously generated or should have generated an unknown opcode message.

Reply:
It is a behavior change but I signed off based on the fact @Pokechu22 said they tested datel and didn't see any side effects.